test/test_flagging.py

```diff
@@ -117,3 +118,19 @@ def test_flagging_no_permission_error_with_flagging_disabled(self):
                 self.fail("launch() raised a PermissionError unexpectedly")
 
         io.close()
+
+
+class TestInterfaceConstructsFlagMethod:
```

Comment:
feels like this tests internal implementation rather than functionality, idk if needed